experimental/packages/otlp-grpc-exporter-base/src/configuration/otlp-grpc-env-configuration.ts

```diff
@@ -128,16 +133,12 @@ function getUrlFromEnv(signalIdentifier: string) {
  * @param signalIdentifier
  */
 function getInsecureSettingFromEnv(signalIdentifier: string): boolean {
-  const signalSpecificInsecureValue = process.env[
+  const signalSpecificInsecureValue = getStringFromEnv(
     `OTEL_EXPORTER_OTLP_${signalIdentifier}_INSECURE`
-  ]
-    ?.toLowerCase()
-    .trim();
-  const nonSignalSpecificInsecureValue = process.env[
+  )?.toLowerCase();
+  const nonSignalSpecificInsecureValue = getStringFromEnv(
     `OTEL_EXPORTER_OTLP_INSECURE`
-  ]
-    ?.toLowerCase()
-    .trim();
+  )?.toLowerCase();
 
   return (
```

Comment:
I think this whole function could be simplified to (untested):

```ts
// Use `getStringFromEnv` because any value, even if invalid, in the signal-specific envvar should win.
return (getStringFromEnv(`OTEL_EXPORTER_OTLP_${signalIdentifier}_INSECURE`)
  ? getBooleanFromEnv(`OTEL_EXPORTER_OTLP_${signalIdentifier}_INSECURE`)
  : getBooleanFromEnv('OTEL_EXPORTER_OTLP_INSECURE')
);
```

And the long comment on block starting with `It will allow the following values as {@code true}` can be removed.